sklearn/impute/_base.py

```diff
@@ -612,7 +612,26 @@ def fit(self, X, y=None):
             raise ValueError("'sparse' has to be a boolean or 'auto'. "
                              "Got {!r} instead.".format(self.sparse))
 
-        self.features_ = self._get_missing_features_info(X)[1]
+        missing_features_info = self._get_missing_features_info(X)
+        self.features_ = missing_features_info[1]
+
+        return missing_features_info
```

Comment:
Why not just return the mask if ``features_`` is already stored as an attribute?

Reply:
@amueller Good point, I'll do it right now.
Thanks.